datasets.md

```diff
@@ -16,7 +16,7 @@
 |[[6]](#6)| Uzbekistan, Tajikistan | 5,302            | Multi-class| CC BY-4.0|
 |[[7]](#7)| Germany           | 2,550             | Multi-class| DL-DE->BY-2.0|
 |[[8]](#8)| Brazil            | 800                   | Multi-class| CC BY-4.0|
-|[[9]](#9)| Tanzania          | 392                   | Multi-class| CC BY-4.0|
+|[[9]](#9)| Tanzania          | 1192                   | Multi-class| CC BY-4.0|
```

Comment:
@adebowaledaniel , I think the new data should be added in a new row since (as far as I know) its not part of the Great African Food Company dataset which is referenced in this row.